program/tests/helpers/mod.rs

```diff
@@ -0,0 +1,7 @@
+#![allow(clippy::arithmetic_side_effects)]
+#![allow(dead_code)]
```

Comment:
Since test banks compile individually, we'll soon otherwise get `dead_code` warnings for any helpers which are not used in every single one.